src/services/item/plugins/publication/published/plugins/search/meilisearch.ts

```diff
@@ -88,7 +97,13 @@ export class MeiliSearchWrapper {
     this.logger = logger;
 
     // create index in the background if it doesn't exist
-    this.getIndex();
+    this.getIndex().then(async () => {
+      // set facetting order to count for tag categories
+      await this.meilisearchClient.index(ACTIVE_INDEX).updateFaceting({
+        maxValuesPerFacet: 50,
```

Comment:
Can this be changed ? Why is it set to 50 ? Does it have a link to pagination ?

Reply:
This is the number of facets you get when requesting faceting. Default value is 10, so you usually receive the 10 most used facets. For Library I'd like to show more than 10, so I increased this number. I will add a comment, close this conversation if it is okay for you.

Reply:
https://www.meilisearch.com/docs/reference/api/settings#faceting-object
I think the exact definition of `facet` is more the **set** of used tag + count.